core/src/test/java/org/frankframework/http/authentication/HttpSenderAuthenticationTest.java

```diff
@@ -504,22 +524,22 @@ void testRetryNonRepeatableMultipartPayloadOnOAuthAuthenticationTokenExpired() t
 		assertEquals("(SocketException) Connection reset", exception.getMessage());
 	}
 
-	@Test
-	void testRetryOnResetAuthenticated() throws Exception {
-		sender.setUrl(getServiceEndpoint() + MockAuthenticatedService.oauthPath);
-		sender.setTokenEndpoint(getTokenEndpoint() + MockTokenServer.PATH);
-		sender.setClientId(MockTokenServer.CLIENT_ID);
-		sender.setClientSecret(MockTokenServer.CLIENT_SECRET);
-		sender.setResultStatusCodeSessionKey(RESULT_STATUS_CODE_SESSIONKEY);
-		sender.setTimeout(100000);
-
-		sender.configure();
-		sender.start();
-
-		tokenServer.setScenarioState(MockTokenServer.SCENARIO_CONNECTION_RESET, MockTokenServer.SCENARIO_STATE_RESET_CONNECTION);
-
-		result = sendMessage();
-		assertEquals("200", session.getString(RESULT_STATUS_CODE_SESSIONKEY));
-		assertNotNull(result.asString());
-	}
+//	@Test
```

Comment:
Uitgecomment? Kan deze test weg of moet dit nog gefixt worden?

Reply:
Oeps!